Supertext.Base.Net/Http/UriBuilder.cs

```diff
@@ -1,36 +1,26 @@
 ﻿using System;
-using Microsoft.AspNetCore.Http;
-using Supertext.Base.Common;
 using Supertext.Base.Http;
 
 namespace Supertext.Base.Net.Http
 {
-    internal class UriBuilder : IUriBuilder
+    internal class UriBuilder : IUriBuilder, IDomainInitializer
     {
-        private readonly IHttpContextAccessor _contextAccessor;
+        private string _currentDomain;
+        private const string DomainPlaceholder = "{domain}";
 
-        public UriBuilder(IHttpContextAccessor contextAccessor)
+        public Uri CreateAbsoluteUri(string relativeUrl)
         {
-            _contextAccessor = contextAccessor;
+            return new Uri($"https://{_currentDomain}/{relativeUrl}");
```

Comment:
I think the before existing solution was better. 
Like: `return new Uri($"https://{_currentDomain}", relativeUrl);`